README.md

```diff
@@ -1,8 +1,14 @@
-# example-typescript-mod
-example eaglerforge mod made using typescript
+# Example TypeScript Mod
+Example EaglerForge mod template.
 
-## how to compile?
-first, run `npm i` to download dependencies and then do `npm run build` and the single js file should be in a folder called "build"
+## How to build?
+WARNING : Bun is experimental on windows
```

Comment:
bun shouldn't be the only option

Reply:
i agree

Reply:
me too

Reply:
> me too

what should we do tho im silly

Reply:
just have npm since bun is backwards compatible with npm if people want to use bun they can